test/processor.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prefer-const */
```

Comment:
Besides the check of the `newStateRoot`, it should check also the `newAccHashInput`